spec/factories/rent.rb

```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
   factory :rent do
+    from    { Faker::Date.today }
+    to      { Faker::Date.forward(2) }
+    book    { 1 }
```

Comment:
you may use: `{ create(:book) }` to ensure the rent always has a valid book instead.

Reply:
Done